### ISSUE TITLE
ToggleButton gets smaller when it's toggled under High Contrast

### ISSUE THREAD

**Issue by StephenLPeters** [microsoft/microsoft-ui-xaml#2938]
copied from internal [issue](https://microsoft.visualstudio.com/OS/_workitems/edit/27534465)

A ToggleButton visually appears to get smaller when you toggle it when High Contrast is enabled.  This happens because it draws a border that's the same color as the background, so the border nearly disappears.  Although due to rounding, there's still some weird halo of fuzzy pixels around the corners.

![image](https://user-images.githubusercontent.com/7758786/87735534-9fe1a180-c78a-11ea-86c0-281c48e02093.png)

This screenshot shows 3 identical ToggleButtons, with the 2nd one toggled. I drew a dotted red alignment line onto the screenshot so it's clear that there's a visual gap below the toggled button.

The problem appears to be that ToggleButtonBorderBrushChecked is not well chosen for the HighContrast theme.  I'd suggest using SystemControlHighlightBaseMediumLowBrush instead.  I tried overriding the brush in my app, and it looks much better:

![image](https://user-images.githubusercontent.com/7758786/87735541-a7a14600-c78a-11ea-8c68-f08f93cfb4b3.png)

![](https://microsoft.visualstudio.com/8d47e068-03c8-4cdc-aa9b-fc6929290322/_apis/wit/attachments/55dd6e10-d857-4073-96c6-e62857d7c721?fileName=image.png)

**Comment by StephenLPeters** [comment]
the black border could be guarding against the toggle button being placed on a background of the same color.

**Comment by jtippet** [comment]
> the black border could be guarding against the toggle button being placed on a background of the same color.

That's possible, although makes for an unfortunate situation of optimizing for the rare bad case, at the expense of the common & good case.  Maybe put a white border around it (on HC Black theme), like the other toggle buttons?  That will keep the visual size of the button consistent with other buttons, while still providing at least some (admittedly poor) contrast in case the button is placed over a teal background.

Here it is with `ButtonBorderThemeBrush` (which is white on HC Black and black on HC White):

![image](https://user-images.githubusercontent.com/10259764/87851316-6dc86080-c8ac-11ea-8582-70fa99eed3fb.png)

It's not completely unprecedented, since here's the Font dialog from Notepad:

![image](https://user-images.githubusercontent.com/10259764/87851339-a2d4b300-c8ac-11ea-92ec-2414f6735bda.png)
